examples/Makefile

```diff
@@ -1,9 +1,17 @@
-JULIAHOME = $(abspath ..)
+JULIAHOME := $(abspath ..)
 include $(JULIAHOME)/Make.inc
 
 FLAGS = -Wall -Wno-strict-aliasing -fno-omit-frame-pointer \
```

Comment:
may as well `:=` here? guess this was missed in the mass replacement of `=` with `:=` in almost all of the makefiles last year